.ui/library/lviProfile.cpx

```diff
@@ -0,0 +1,159 @@
+{
+	"components": [
+		{
+			"className": ".sf-listViewItem .lviProfile",
+			"hiddenComponent": true,
+			"id": "21e4-9a74-e568-74d9",
+			"props": {
+				"children": [
+					"bdc9-35c2-1456-d347"
+				],
+				"name": "lviProfile",
+				"parent": "57f4-201f-4bfc-5fc6"
+			},
+			"source": {
+				"page": "__library__"
+			},
+			"type": "ListViewItem",
+			"userProps": {
+				"flexProps": {},
+				"left": "0",
+				"top": "118.97"
+			}
+		},
+		{
+			"className": ".sf-flexLayout .flAccountUser",
```

Comment:
"type": "FlexLayout" but uses a component class. If its a component use the component not the component's class only.

Reply:
done.